.github/workflows/build-release-zip-file.yml

```diff
@@ -6,13 +6,109 @@ on:
         description: 'By default the zip file is generated from the branch the workflow runs from, but you can specify an explicit reference to use instead here (e.g. refs/tags/tag_name or refs/heads/release/x.x). The resulting file will be available as an artifact on the workflow run.'
         required: false
         default: ''
+      skip_verify:
+        description: 'Skip the PR verification step (default: false) pass true to skip'
+        required: false
+        default: 'false'
 
 permissions: {}
 
 jobs:
+  verify:
+    name: 'Verify if any PR is left open by author:app/github-actions or with "cherry" in the title'
+    outputs:
+      runBuildZipJob: ${{ steps.verify-prs.outputs.runBuildZipJob }}
+    runs-on: ubuntu-20.04
+    steps:
+        - name: Verify if any PR is left open by author:app/github-actions or with "cherry" in the title
```

Comment:
I was wondering if we should add a similar check during the build changelog step, or that would be over doing things? Since this is the most critical step, this check is fine here.

Reply:
I think I'd err on the side of *not* repeating the check 🤔